tools/c7n_azure/setup.py

```diff
@@ -67,6 +67,7 @@
                       "azure-mgmt-subscription",
                       "azure-mgmt-eventgrid>=2.0.0rc1",
                       "azure-graphrbac",
+                      "azure-keyvault",
```

Comment:
Why do we need this and azure-mgmt-keyvault?

Reply:
I think azure-mgmt-keyvault can only perform CRUD with KV resource.. But we need azure-keyvault for operations with keys.

Reply:
@logachev is right. I would prefer to use one client rather than 2 different clients but that's how the sdk is providing access to resources right now.